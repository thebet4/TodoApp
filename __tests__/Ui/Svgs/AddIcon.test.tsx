import renderer from 'react-test-renderer';

const AddIcon = require('../../../src/Ui/Sgvs/AddIcon').default

describe('<AddIcon />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(AddIcon).toJSON()
    expect(tree).toMatchSnapshot()
  });
});
