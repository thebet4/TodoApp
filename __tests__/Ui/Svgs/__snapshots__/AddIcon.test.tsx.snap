// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddIcon /> renders correctly 1`] = `null`;
